python/ck4inductor/universal_gemm/gen_instances.py

```diff
@@ -3,8 +3,10 @@
 
 import logging
 import os
+import re
 import subprocess
-from dataclasses import replace
+
```

Comment:
also for the whitespace changes, are they coming from a linter or were they introduced manually?